OrbitGl/TimeGraph.cpp

```diff
@@ -683,11 +683,18 @@ void TimeGraph::UpdatePrimitives(bool a_Picking) {
 
                 textBox.SetText(text);
               } else if (timer.m_Type == Timer::INTROSPECTION) {
-                textBox.SetText(
-                    string_manager_->Get(timer.m_UserData[0]).value_or(""));
+                std::string text = absl::StrFormat(
+                    "%s %s",
+                    string_manager_->Get(timer.m_UserData[0]).value_or(""),
+                    time.c_str());
+                textBox.SetText(text);
               } else if (timer.m_Type == Timer::GPU_ACTIVITY) {
-                textBox.SetText(
-                    string_manager_->Get(timer.m_UserData[0]).value_or(""));
+                std::string text = absl::StrFormat(
+                    "%s; submitter tid: %d  %s",
```

Comment:
Would removing "tid" make the string look better or would it be less comprehensible? (Just to shave off a few characters.)

Reply:
Yeah, should have mentioned in the PR description that this is functional, but not pretty. Not sure what is best here. I think it's pretty clear from context that the submitter is a thread id, so I'd be happy to remove the "tid" part. 

What I probably want in the end is minimal strings in the timeslice itself but more info in the tooltip / on hover.

Reply:
Removed "tid" from the string.